Scottish-Tech-Army/Volunteer-app
# Cleanup gitignore - readme fixes

- [Description](#description)
  - [Type of change](#type-of-change)
- [Testing locally](#testing-locally)
- [Checklist](#checklist)

# Description

- added .expo to the .gitignore files as it's not intended to be checked in 
- fixed a layout issue with readme.md

## Type of change

Please delete options that are not relevant

- Cleanup task

# Testing locally

- Open readme.md file and see if the numbering is displayed correctly in markdown view
- make changes in .expo files and see if it's shown in the git changes


# Checklist

- no code changes at all and only adding not needed .expo directory to gitignore.
